src/ScoutElasticServiceProvider.php

```diff
@@ -78,4 +83,9 @@ public function register()
                 return ClientBuilder::fromConfig($config);
             });
     }
+
+    protected function isLumen()
+    {
+        return Str::contains($this->app->version(), 'Lumen');
```

Comment:
Please, change this check to:

    return $this->app() instanceof Laravel\Lumen\Application;

Because it's a lot faster then check the string (yeah, I'm crazy):

Benchmark for 9999999 repeations:

* Laravel - using Str::contains(): 6.24s (436% slower)
* Laravel - using instanceof: 1.43s
* Lumen - using Str::contains(): 4.85s (664% slower)
* Lumen - using instanceof: 0.73s